database/nakadi/10_data/00_create_schema.sql

```diff
@@ -0,0 +1,18 @@
+CREATE ROLE zalando_nakadi_data_owner;
```

Comment:
as far as I remember this is a very outdates way of creating of the roles. 

ACID team has much more advanced and clean code for that.